src/App.tsx

```diff
@@ -1,60 +1,65 @@
-import classNames from 'classnames';
-
+import React, { useState, useEffect } from 'react';
+import './App.scss';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
-
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
-import { UserSelector } from './components/UserSelector';
-import { Loader } from './components/Loader';
+import { getAllUsers } from './api/api';
```

Comment:
This is not a typo, but it will be much easier to understand what you have in the api.ts file if you name it accordingly, like the other files in the api folder (posts.ts, comments.ts), so it would be logical to call it users.ts.